sds_data_manager/stacks/processing_stack.py

```diff
@@ -87,30 +86,21 @@ def __init__(
             account_name=account_name,
```

Comment:
I have been wondering if we can delete this whole file and start fresh? meaning de-couple how we create resources. We would still same thing as what we do in this file. But we create batch job, lambda and event rule separately. Create resources by logics of what goes together.

Reply:
Yes. Especially with event rule. Let's create another ticket for organization.